docs/products/sessions/README.md

```diff
@@ -49,3 +49,15 @@ The equivalent CLI command:
 grid session delete $INTERACTIVE_NODE_ID
 ```
 
+## Share Data Between Sessions
+At this time the only supported way to share data between sessions is to mount the sessions locally and copy files to the local mounts. Establishing SSH connection to your sessions is a prerequisite to this. See these [steps](https://github.com/gridai/grid-docs/blob/doc-182-share-data-between-sessions/docs/products/sessions/how-to-ssh-into-a-session.md) to set up SSH connection with your sessions. **Be sure to logout of the interactive session before attempting the next step**. After that you can do the following to create your session mounts:
+
+```
+# Do for each session you wish to share data between
+mkdir <dir for each session mount>
+grid session mount <your session> <created session mount dir>
+mkdir <created session mount dir>/shared_data
+
+# to share data from a session to another session
+cp -R <dir of interest in created session mount dir> <other session mount dir>/shared_data
+```
```

Comment:
@oojo12 I like how you used the concepts of Session A and Session B in your testing. It makes it clear and easy to use the commands. Could you incorporate that here?

Reply:
I kinda wanted to stay away from that here and use more general language. @essiequoi if you still would like the concept of Session A Session B here let me know and I will make that change.

Reply:
`<dir of interest in created session mount dir> ` is unclear. you could also use the language `source` and `target` (or `destination` to distinguish between the two sessions